tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>why is<strong> </strong><code>apply_clustering_to_first</code> <strong>  </strong>named as such?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>How should I improve it? The version I updated also has the name <code>apply_clustering_to_dense</code>.  </p>

Reply:
<p>The new <code>apply_clustering_to_dense</code> name seems good to me!</p>

Reply:
Sorry, I am confused. What was your thought on `apply_clustering_to_first`?

Reply:
I just didn't understand what "first" stood for in the name. Because it looked like it just applies clustering to the layer if it's in the list of layers to be clustered and leaves it otherwise. So I just didn't quite understand what the 'first' referred to.